Assets/VRTK/Editor/VRTK_ObjectSetup.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+    using UnityEditor;
+    using VRTK.GrabAttachMechanics;
+    using VRTK.SecondaryControllerGrabActions;
+
+    public class VRTK_ObjectSetup : EditorWindow
+    {
+        private enum PrimaryGrab
+        {
+            ChildOfController,
+            FixedJoint,
+            Climbable,
+            CustomJoint,
+            RotatorTrack,
+            SpringJoint,
+            TrackObject
+        }
+        private enum SecondaryGrab
+        {
+            SwapControllers,
+            ControlDirection,
+            AxisScale
+        }
+        private bool useGrab = true;
+        private bool holdGrab = false;
+        private bool useUse = false;
+        private bool useIfGrabbed = false;
+        private bool holdUse = false;
+        private PrimaryGrab primGrab;
+        private SecondaryGrab secGrab;
+        private bool disableIdle = true;
+        private bool addrb = true;
+
+        [MenuItem("Window/VRTK/Setup Interactable Object")]
+        private static void Init()
+        {
+            VRTK_ObjectSetup window = (VRTK_ObjectSetup)EditorWindow.GetWindow(typeof(VRTK_ObjectSetup));
+            
+            window.minSize = new Vector2( 300f, 300f );
+            window.maxSize = new Vector2( 400f, 300f );
+            
+            window.autoRepaintOnSceneChange = true;
+            window.titleContent.text = "Setup Object";
```

Comment:
worth naming it "Setup Interactable Object" to make it more obvious what the tab is?